src/doc/src/reference/unstable.md

```diff
@@ -1233,63 +1233,149 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
 
-The `workspace-inheritance` feature allows workspace members to inherit fields
-and dependencies from a workspace.
+### The `workspace.package` table
 
-Example 1:
+*Stabilization*: This would be in [`workspaces.md`][workspaces], under
+[The `workspace.metadata` table][workspace-metadata-table]
 
-```toml
-# in workspace's Cargo.toml
-[workspace.dependencies]
-log = "0.3.1"
-log2 = { version = "2.0.0", package = "log" }
-serde = { git = 'https://github.com/serde-rs/serde' }
-wasm-bindgen-cli = { path = "crates/cli" }
-```
+The `workspace.package` table is where you define keys that can be
+inherited by members of a workspace. These keys can be inherited by
+defining them in the member package with `{key}.workspace = true`.
 
-```toml
-# in a workspace member's Cargo.toml
-[dependencies]
-log.workspace = true
-log2.workspace = true
-```
+Keys that are supported:
 
-Example 2:
+|                |                 |
+|----------------|-----------------|
+| `authors`      | `categories`    |
+| `description`  | `documentation` |
+| `edition`      | `exclude`       |
+| `homepage`     | `include`       |
+| `keywords`     | `license`       |
+| `license-file` | `publish`       |
+| `readme`       | `repository`    |
+| `rust-version` | `version`       |
+
+- `license-file` and `readme` are relative to the workspace root
+- `include` and `exclude` are relative to your package root
+
+Example:
 ```toml
-# in workspace's Cargo.toml
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
+
 [workspace.package]
 version = "1.2.3"
 authors = ["Nice Folks"]
 description = "..."
 documentation = "https://example.github.io/example"
-readme = "README.md"
-homepage = "https://example.com"
-repository = "https://github.com/example/example"
-license = "MIT"
-license-file = "./LICENSE"
-keywords = ["cli"]
-categories = ["development-tools"]
-publish = false
-edition = "2018"
 ```
 
 ```toml
-# in a workspace member's Cargo.toml
+# [PROGECT_DIR]/bar/Cargo.toml
 [package]
+name = "bar"
 version.workspace = true
 authors.workspace = true
 description.workspace = true
 documentation.workspace = true
-readme.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
-license-file.workspace = true
-keywords.workspace = true
-categories.workspace = true
-publish.workspace = true
 ```
 
+
+### The `workspace.dependencies` table
+
+The `workspace.dependencies` table is where you define dependencies to be
+inherited by members of a workspace. 
+
+To inherit a dependency:
+- [specify the dependency][specifying-dependencies] in this table
+  - Dependencies from this table cannot be declared as `optional`
+  - [`features`][features] declared in this table are additive with the `features` from `[dependencies]`
+- [inherit that dependency][inheriting-a-dependency-from-a-workspace] in the member
+`[dependencies]` table.
```

Comment:
I feel like the outer bullet list loses the association of the two steps

What about
```
> Specifying a workspace dependency is similar to [package dependencies][...] except:
> - ...
> - ...
> You can then [inherit the workspace dependency as a package dependency][...]
```

Reply:
That is much better!